--- /cvs/drupal/contributions/modules/cck/modules/userreference/Attic/userreference.install	2009/08/11 20:39:33	1.23.2.9
+++ /cvs/drupal/contributions/modules/cck/modules/userreference/Attic/userreference.install	2009/11/07 11:04:15	1.23.2.10
@@ -1,5 +1,5 @@
 <?php
-// $Id: userreference.install,v 1.23.2.8 2009/08/10 04:19:46 markuspetrux Exp $
+// $Id: userreference.install,v 1.23.2.9 2009/08/11 20:39:33 markuspetrux Exp $
 
 /**
  * @file
@@ -124,6 +124,8 @@
 function userreference_update_6002() {
   $ret = array();
 
+  drupal_load('module', 'content');
+
   $result = db_query("SELECT field_name, global_settings FROM {". content_field_tablename() ."} WHERE type = 'userreference'");
   while ($userreference = db_fetch_object($result)) {
     $global_settings = unserialize($userreference->global_settings);
